zap/src/main/java/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -499,6 +501,21 @@ protected boolean nodeInScope(String nodeName) {
         return parentScanner.isInScope(nodeName);
     }
 
+    private FilterResult filterNode(StructuralNode node) {
+        for (ScanFilter scanFilter : parentScanner.getScanFilters()) {
+            try {
+                FilterResult filterResult = scanFilter.isFiltered(node);
+                if (filterResult.isFiltered()) {
+                    return filterResult;
+                }
+            } catch (Exception ex) {
+                // Swallowing exception to prevent filter from breaking scanner.
```

Comment:
The comment could be removed (that's the expected when catching `Exception`).

Reply:
sure. will do that.